README.md

```diff
@@ -139,6 +139,47 @@ lol
 //=> array with the descriptions of the mastery "Precision"
 ```
 
+#### `getSummoner()`
+
+```js
+lol.getSummoners().then(summoners => {
+  console.log(summoners);
+});
+//=> array of summoner spells
+```
+
+#### `getSummoner()`
+
+```js
+lol
+  .getSummoner({
+    summoner: "Barrier"
```

Comment:
If `summoner` is the only property of this object, then the argument can be a string instead of an object with only one property.